docs/understanding-tracking-design/managing-event-specifications/ui/index.md

```diff
@@ -0,0 +1,89 @@
+sidebar_label: "Using the UI"
+sidebar_position: 1
+sidebar_custom_props:
+  offerings:
+    - enterprise
+    - cloud
+---
+
+## Crafting a New Event Specification through the Console
+
+Event specifications serve as direct counterparts to [data structures](/docs/understanding-tracking-design/managing-your-data-structures/ui/index.md) and encapsulate the documented events within a data product.
+
+:::info
+Please note that the creation of event specifications is exclusive to data products.
+:::
+
+Create event specifications in tandem with the latest deployed data structure version in production. This ensures tracking implementation instructions align with the validation criteria in the event specification.
+
+Your event specification guarantees compatibility with the associated data structure version. This ensures events tracked using specified instructions pass validation for the associated data structure.
+
+If a newer event specification version is deployed, consider upgrading event specification and tracking to the latest data structure, ensuring ongoing compliance with validation requirements.
+
+## Creating and Editing Event Specifications
+
+To create a new event specification, follow these steps:
+
+1. Select a data product
+2. Click the "Create event" button
+3. A dialog will appear, prompting you to enter a name for your event specification and click "Save and continue"
+4. Your first event specification will be displayed on the page
+
+![Create an Event Specification](images/create-event-specification.png)
+
+To add more information or modify an existing event specification, follow these steps:
+
+1. Navigate to the appropriate data product
+2. Select the desired event specification
+3. This action will open on overview of the selected event specification containing the details that have been added to date.
+
+This interface is divided into focused sections; explore each section below for more details.
+
+![Example of an Event Specification Overview](images/event-specification-overview.png)
+
+### Event Information
+
+This section provides essential meta-information for your event specification, including the event name, description, and the applications in which this event is tracked.
+
+### Event Data Structure
+
+This section defines the structure of your data and specifies the types of properties associated with this event, along with its version.
+
+You can choose from two types of data structures:
+
+- **Standard**; Provided by the Snowplow tracker by default
+- **Custom**; Provided by your organization
+
+![Event Data Structure](images/event-data-structure.png)
+
+## Entity Data Structures
+
+Within this section, you have the flexibility to choose the specific entities that should be associated with the event when it is triggered.
+
+To facilitate making an informed selection of your entities, you are able to view a detailed breakdown of the properties associated with your chosen entity for a specific version.
```

Comment:
You can view a detailed breakdown of the properties associated with the chosen entities (and its version) by ....

Reply:
done